### ISSUE TITLE
Error when using get_scatt_map from COSILike

### ISSUE THREAD

**Issue by hiyoneda** [cositools/cosipy#262]
An error occurred when running the Crab spectral fitting notebook with the develop branch (f3a7477b726a69e334bf250f6ee01220f1dfc91c).
Probably, after some updates,`self._sc_orientation.get_scatt_map` needs `target_coord` as its input parameter now.

![Screenshot 2024-11-11 at 17 40 08](https://github.com/user-attachments/assets/47ce55ba-efe8-44dd-98a9-5f167f2224e4)

**Comment by ckarwin** [comment]
@hiyoneda Yup, that's right. I can update this and make a PR. Can you check it when it's ready?

**Comment by hiyoneda** [comment]
> @hiyoneda Yup, that's right. I can update this and make a PR. Can you check it when it's ready?

yes, I'll check it.

**Comment by israelmcmc** [comment]
Closed by #264
